platform/linux-generic/pool/subsystem.c

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2017, ARM Limited. All rights reserved.
+ *
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+#include <odp_module.h>
+
+#define SUBSYSTEM_VERSION 0x00010000UL
```

Comment:
OK, so you're saying that this is unique to each module and could just as easily be specified directly in the `ODP_SUBSYSTEM_DEFINE()` macro below?  So how is this used to detect version mismatches? Or is it purely for documentation purposes at this stage? If we're supporting separate loadable modules then it would seem some type of load-time version compatibility check is needed.